Sources/Tracing/Tracer.swift

```diff
@@ -127,3 +127,65 @@ extension Tracer {
         }
     }
 }
+
+#if compiler(>=5.5) // we cannot write this on one line with `&&` because Swift 5.0 doesn't like it...
```

Comment:
oh, lol okey :)

Reply:
Copied directly from @weissi's NIO async/await support 😁

Reply:
do you support Swift 5.0 in this repo? If no, then you can remove this extra line, from Swift 5.1 on `#if compiler(>=5.5) && $AsyncAwait` works as expected. In Swift 5.0 that line will fail compilation and error (think it complains about something that has to be numeric or so).

Reply:
For better or worse we currently do support 5.0, yes.

Though to be honest we'll drop it all and require async soon I guess...?
But for now yes, we do.

Reply:
Yeah, I don't think we need to maintain backwards compatibility here.

Reply:
Given that this repo isn't 1.0.0 tagged, I'd recommend dropping all old Swift versions :P

Reply:
Yeah, we'll do that.

The discussions last year during gsoc were on the dramatic side with people requesting 5.0... 😉 

Now as we want to require task-locals we'll rehash the lib and require async features anyway and drop the 5.0, not much to be discussed about it this year about it.